.github/workflows/composite/benchmark/action.yml

```diff
@@ -0,0 +1,71 @@
+# Copyright 2022 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+name: 'Run Benchmark'
+description: 'Composite action to Run Benchmark'
+inputs:
+  api_count:
+    description: 'api count'
+    required: true
+    default: 10
+  version_count:
+    description: 'version count'
+    required: true
+    default: 1
+  dbconnection:
+    description: 'Database connection string'
+    required: false
+    default: 'host=localhost port=5432 user=postgres dbname=registry_tester password=postgres sslmode=disable'
+runs:
+  using: "composite"
+  steps:
+    - name: Set up Go 1.x
+      uses: actions/setup-go@v2
+      with:
+        go-version: ^1.17
+
+    - name: Install
+      run: go install ./...
+      shell: bash
+
+    - name: Start the Registry Server with Postgres
+      run: registry-server &
+      shell: bash
+      env:
+        REGISTRY_DATABASE_CONFIG: ${{ inputs.dbconnection }}
```

Comment:
When you run the server without a config file it uses a [default configuration](https://github.com/apigee/registry/blob/40b8c96b63f88fd908eb77f98b919eab0f8a9370/cmd/registry-server/main.go#L76) that connects to a SQLite database. If we need this to run against a different database you will need to create a config file that references this environment variable.

Reply:
This is a template action which has all the steps for benchmark tests.
I found that running the test against sqlite was not reporting accurate results. 
If you look at benchmark.yml you will see the postgres DB.